### ISSUE TITLE
CS1591 code fix should offer "<inheritdoc />" for explicit static interface method implementations

### ISSUE THREAD

**Issue by AArnott** [dotnet/roslyn#70456]
Given the following 

```cs
public interface IFoo
{
	/// <summary>
	/// My decoding method.
	/// </summary>
	static abstract bool TryDecode();
}

public class Foo : IFoo
{
	public static bool TryDecode() => true;
}
```

Placing the caret on the `Foo.TryDecode`  method and activating the lightbulb offers a code fix to add `<inheritdoc/>`:
![image](https://github.com/dotnet/roslyn/assets/3548/6cf9c1fe-98f3-4d24-942d-bec562676fe6)

But switch it up just a little, by making the method an *explicit* interface implementation, and the code fix is no longer offered:
![image](https://github.com/dotnet/roslyn/assets/3548/2ea68d2b-008f-44bd-8fd1-cd5a375d563f)

If I make the code fix manually, the compiler allows it (though [stylecop doesn't yet like it](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3595)).

So, to satisfy the analyzers that want public members explicitly documented, such as SA1600, can the automated code fix be offered consistently across interface implementations?

**Comment by CyrusNajmabadi** [comment]
I'm a little confused by this. Explicit impl members are not public. So what is complaining and saying they should have doc comments?
